scripts/test_all_fuzzers.sh

```diff
@@ -32,12 +45,16 @@ do
 	    echo "[+] Done testing $fuzzer"
     else
         echo "[*] Building $fuzzer"
+        cargo sweep -s # record cargo action and then clean output
```

Comment:
This should be done for `cargo make test` as well (or rather, for this whole loop execution)